script/receiver

```diff
@@ -1,13 +1,15 @@
 #!/usr/bin/env ruby
 # frozen_string_literal: true
+
```

Comment:
Should this be renamed to `drain`, too? 

I personally like having such files live in `bin`, since I feel `script` is a pretty Rails'y tradition. I feel this file represents a full app, not just a "script". Then again, in other worlds (such as PHP?) everything is a "script". Matter of taste, I guess.